tests/vmcheck/test-cached-rpm-diffs.sh

```diff
@@ -58,6 +58,33 @@ call_dbus() {
     -m org.projectatomic.rpmostree1.OS.$method "$@"
 }
 
+if vm_cmd grep -q 'ID=.*centos' /etc/os-release; then
```

Comment:
That'll blow up if we ever ran our installed tests on rhel.  How about `grep -q ID_LIKE=.*rhel`?

Reply:
Thanks, that's much nicer!